thoth-tech/doubtfire-web
# refactor: upload files

# Description
Migrates the grade-icon from CoffeeScript to TypeScript as a visual indicator for grading within the application. It provides a user interface for displaying a grade with an associated tooltip, making it easier for tutors to quickly assess and review student performance.

Before: 
![Screenshot 2024-08-08 at 10 33 17 PM](https://github.com/user-attachments/assets/c43d491c-5058-495c-9664-9155c0a8a9dd)
<img width="1680" alt="Screenshot 2024-08-08 at 9 59 17 PM" src="https://github.com/user-attachments/assets/4320d585-42ab-4a0d-b43b-80ab1414a1f8">

After:
![Screenshot 2024-08-08 at 9 59 27 PM](https://github.com/user-attachments/assets/2dd371ef-6bb7-43aa-a72b-ff90bfc23eaf)
![Screenshot 2024-08-08 at 9 58 57 PM](https://github.com/user-attachments/assets/c759da13-472c-4c46-8ace-9432d2c4bba8)

Fixes # (issue)

## Type of change


# How Has This Been Tested?
1. Login as student and create portfolio
Select the target grade, the grade icon appears properly.

2. Login as tutor and select student list
The grade icon appears properly according to the student's target grade.

## Testing Checklist:


# Checklist:
